osquery/tables/system/windows/signature.cpp

```diff
@@ -0,0 +1,238 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <codecvt>
+#include <map>
+#include <string>
+
+// clang-format off
+#define _WIN32_DCOM
+#define WIN32_LEAN_AND_MEAN
+
+#include <Windows.h>
+#include <psapi.h>
+#include <stdlib.h>
+#include <tlhelp32.h>
+#include <wincrypt.h>
+#include <Softpub.h>
+// clang-format on
+
+#include <osquery/core.h>
+#include <osquery/core/conversions.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/tables.h>
+
+namespace osquery {
+struct SignatureInformation final {
+  enum class Result { Valid, Trusted, Invalid, Missing, Distrusted, Untrusted };
+
+  std::string path;
+  Result result;
+};
+
+Row& operator<<(Row& row, const SignatureInformation& signature_info) {
```

Comment:
While this is pretty awesome how you're making use of the `<<` on `Row`s, I'm honestly not sure how to feel about it as it seems dangerous to overload an operator on a data structure that gets used in so many places in this high of namespace - /CC @theopolis to potentially put my worries to rest, but are there potentially other (arguably less awesome) ways to achieve the same behavior?